### ISSUE TITLE
Add Pagination

### ISSUE THREAD

**Issue by RobLoach** [RobLoach/love-lutris#3]
If you have like a million games you can't list them all.

**Comment by henriquess1995** [comment]
This question would be very interesting to be corrected, even more now with proton project, DXVK etc hundreds of games are compatible in Lutris.

**Comment by RobLoach** [comment]
Yeah! Would be great to get scrolling working on the list. Either that or adopt a GUI library to manage the list for us.
